From fb584589f707853d85a081c99b1b82598c2631c1 Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Thu, 24 Nov 2022 15:10:27 -0700
Subject: [PATCH] Don't link tensorflow-lite against gemmlowp

We can't link against gemmlowp as it is a header only library.

Fixes:
/bin/ld: cannot find -lgemmlowp: No such file or directory

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[Upstream status:
https://github.com/tensorflow/tensorflow/pull/58678]
---
 tensorflow/lite/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index f9c30d6a046..3fe5ed15605 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -560,7 +560,6 @@ target_link_libraries(tensorflow-lite
     farmhash
     fft2d_fftsg2d
     flatbuffers::flatbuffers
-    gemmlowp
     ruy::ruy
     pthreadpool
     ${CMAKE_DL_LIBS}
-- 
2.34.1

